documentation/src/content/docs/en/essentials/manage-states.mdx

```diff
@@ -0,0 +1,240 @@
+import TabItem from "../../../../components/Tabs/TabItem.astro";
+
+# State Management
+
+State in effector is managed through stores - special objects that hold values and update them when receiving events. Stores are created using the [createStore](/api/effector/createStore) function.
+
+## Store Creation (#store-creation)
+
+```ts
+import { createStore } from "effector";
+
+// Create store with initial value
+const $counter = createStore(0);
+// with explicit typing
+const $user = createStore<{ name: "Bob"; age: 25 } | null>(null);
+const $posts = createStore<Post[]>([]);
+```
+
+:::tip{title="Store naming"}
+In effector it's conventional to use the `$` prefix for stores. This helps distinguish them from other entities and improves code readability.
+:::
+
+## Reading Values (#read-store-value)
+
+There are several ways to get the current value of a store:
+
+1. getState() method
+
+```ts
+console.log($counter.getState()); // 0
+```
+
+2. Subscribe to changes via watch
+
+```ts
+$counter.watch((value) => {
+  console.log("Counter changed:", value);
+});
+```
+
+3. Using framework integration hooks
+
+<Tabs>
+  <TabItem label="React">
+
+```ts
+import { useStore } from 'effector-react'
+import { $counter } from './model.js'
+
+const Counter = () => {
+  const counter = useUnit($counter)
+
+  return <div>{counter}</div>
+}
+```
+
+  </TabItem>
+  <TabItem label="Vue">
+
+```html
+<script setup>
+  import { useUnit } from "effector-vue/composition";
+  import { $counter } from "./model.js";
+
+  const counter = useUnit($counter);
+</script>
+```
+
+  </TabItem>
+  <TabItem label="Solid">
+
+```ts
+import { useStore } from 'effector-solid'
+import { $counter } from './model.js'
+
+const Counter = () => {
+  const counter = useUnit($counter)
+
+  return <div>{counter()}</div>
+}
+```
+
+  </TabItem>
+</Tabs>
+
+## Store Updates (#store-updates)
+
+### Updating via Events (#update-store-via-events)
+
+The simplest way to update a store is to bind it to an event:
+
+```ts
+import { createStore, createEvent } from "effector";
+
+const incremented = createEvent();
+const resetCounter = createEvent();
+
+const $counter = createStore(0)
+  // Increase value by 1 each time the event is called
+  .on(incremented, (state) => state + 1)
+  // Reset value to 0
+  .reset(resetCounter);
+
+$counter.watch((state) => console.log(state));
+
+// Usage
+incremented();
+incremented();
+
+resetCounter();
+
+// Console output
+// 0 - output on initialization
+// 1
+// 2
+// 0
+```
+
+### Updating with Event parameters (#update-stores-with-event-params)
+
+You can update a store using event parameters by passing data to the event like a regular function and using it in the handler.
+
+```ts
+import { createStore, createEvent } from "effector";
+
+const userUpdated = createEvent<{ name: string }>();
+
+const $user = createStore({ name: "Bob" });
+
+$user.on(userUpdated, (state, update) => ({
+  ...state,
+  ...update,
+}));
```

Comment:
```suggestion
$user.on(userUpdated, (user, changes) => ({
  ...user,
  ...changes,
}));
```

Reply:
wouldn't it be better to replace here with updatedValues ?